README.md

```diff
@@ -34,32 +35,31 @@ The main parameters are `--input` as defined above and `--output` for specifying
 
 In order to customize metadata headers, the parameters `--metadata_1_header` through `--metadata_8_header` may be specified. These parameters are used to re-name the headers in the final metadata table from the defaults (e.g., rename `metadata_1` to `country`).
 
-## Distance Method and Thresholds
+## Distance Threshold
 
-The Genomic Address Service Clustering workflow can use two distance methods: Hamming or scaled.
+A distance threshold parameter may be used to constrain the maximum distances between reported sample pairs in the final reports. This can be accomplished by specifying `--threshold DISTANCE`, where `DISTANCE` is a non-negative integer when using Hamming distances or a float between [0.0, 1.0] when using scaled distances. See below for more information on these distance methods.
```

Comment:
For scaled distances, the units are as a percentage, so the range is `[0.0, 100.0]` rather than `[0.0, 1.0]`.

Reply:
Woops, good catch!